edx_sga/tests/integration_tests.py

```diff
@@ -509,32 +520,44 @@ def test_download_annotated(self):
 
     def test_staff_download(self):
         """
-        Test download for staff.
+        Test download for staff. validate each student assignment
```

Comment:
Can you move the period to the end? This should read `Test download for staff and validate each student assignment.`